src/main/kotlin/com/atlassian/performance/tools/infrastructure/Pidstat.kt

```diff
@@ -0,0 +1,35 @@
+package com.atlassian.performance.tools.infrastructure
```

Comment:
Consistent with other `OsMetric`s:
```suggestion
package com.atlassian.performance.tools.infrastructure.os
```

Reply:
Those are already spread out.
- `com.atlassian.performance.tools.infrastructure.api.os.Vmstat`
- `com.atlassian.performance.tools.infrastructure.Iostat`
I moved `Iostat` and `Pidstat` to `.os` as it seems to be a better fit.

Reply:
Ah right, thanks!